src/styles/main.css

```diff
@@ -1,14 +1,34 @@
-:root {
-  --color-primary: #fbe1a6;
-  --color-secondary: #402b1a;
-  --color-accent: #ef9d00;
+* {
+  box-sizing: border-box;
 }
 
 body {
   background-color: var(--color-primary);
   color: var(--color-secondary);
+  margin: 0px;
 }
 
+/* Typography */
+
+.heading {
+  font-family: "Cooper Black", serif;
```

Comment:
Are we able to use our variables here?  Also, maybe these would be good to examples of in the project's Storybook?  Maybe just make a file called "styleguide.mdx" in _stories/_ and just have some examples of these classes being used?

Reply:
How do I document it here?

Reply:
I guess for now I would start by maybe making another couple files in the stories directory
- colors
- typography

And just some examples that way.  Storybook should already be pulling in those main styles so maybe even if it's stating what fonts and hex values we're using?  Doesn't have to go too into depth, just keeping a record of our work done so far will be a great start and help!